src/timepicker-select/timepicker-select.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { TimePickerSelectModule } from "./timepicker-select.module";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { ExperimentalComponenent } from "../../.storybook/experimental.component";
+import { ExperimentalModule } from "..";
+
+storiesOf("Time Picker Select", module)
+	.addDecorator(
+		moduleMetadata({
+			declarations: [ExperimentalComponenent],
+			imports: [
+				TimePickerSelectModule,
+				ExperimentalModule
+			]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("Simple", () => ({
+		template: `
+		<app-experimental-component></app-experimental-component>
+		<div class="bx--form-item">
+			<div class="bx--time-picker">
+				<ibm-timepicker-select display="inline">
```

Comment:
is `display="inline"` the default? Or only option? Should this be something we're handling on the component side?